pumpAlerts.py

```diff
@@ -398,13 +403,12 @@ def checkNewListings(data_t):
         asset = getPercentageChange(asset)
     topPumpDump(tpda_last_trigger, full_data)  # Triggers check for top_pump_dump
 
-    if config["debug"]:
-        print(
-            "Extract time: ",
-            time.time() - start_time,
-            "/ Time ran: ",
-            datetime.datetime.now() - init_dateTime,
-        )
+    logger.debug(
+        "Extract time: %s / Time ran: %s.",
+        time.time() - start_time,
+        datetime.datetime.now() - init_dateTime,
+    )
```

Comment:
Considering this will be sent out at every '1s' interval, would it possibly end up clogging up the debug logs? Do we want to consider putting this under a lower priority group for logs?

Reply:
That's why I set the level to debug and in the confix.yml changed the debug parameter to false. So you will have it turned on only while debugging.